django_celery_beat/migrations/0016_crontabschedule_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-12-15 13:06
```

Comment:
can you use django 3.2.x to regenerate this

Reply:
Re-created with 3.2

Reply:
@trianglesis  It doesn't seem necessary to do the `if` twice. I'll checkout your branch in my projects.

Reply:
looks good